pkg/platform/kube/platform.go

```diff
@@ -466,6 +468,64 @@ func (p Platform) EnrichFunctionConfig(ctx context.Context, functionConfig *func
 		functionConfig.Spec.PriorityClassName = p.Config.Kube.DefaultFunctionPriorityClassName
 	}
 
+	if p.Config.Kube.PreemptibleNodes != nil {
+
+		// we do such stuff to allow exposing features before they are exposed on UI
+		if preemptionMode, exists := functionConfig.Meta.Annotations["custom.nuclio.io/preemptible-mode"]; exists {
+			p.Logger.DebugWithCtx(ctx,
+				"Enriching function preemption mode from function annotations",
+				"preemptionMode", preemptionMode)
+			functionConfig.Spec.PreemptionMode = functionconfig.RunOnPreemptibleNodeMode(preemptionMode)
+		}
+
+		if functionConfig.Spec.PreemptionMode == functionconfig.RunOnPreemptibleNodesAllow {
```

Comment:
Do we support other modes or just ignore other labels for now?

Reply:
we will probably add support for `always` / `only` with node affinity in the future so leaving that as an option